### ISSUE TITLE
Accessibility: input wrong autocomplete value

### ISSUE THREAD

**Issue by SteffleCheff** [ng-select/ng-select#2559]
### Describe the bug

By running the accessibility-checker on my website i noticed an issue with the input-field of the ng-select:
The input-field of the ng-select automatically gets the dropDownId assigned, but this is an invalid value.
This value is used to let the browser know how to autofill user-personal data. The values that can be used are limited and described here: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete#token_list_tokens

So using the dropDownId is an invalid value and this leads to accessibility-issues:
"autocomplete attribute must be used correctly."

See ng-select doc page:
![Image](https://github.com/user-attachments/assets/779a7b2d-40a5-4b92-bcc1-5ab03e415bdf)

There already was an issue for that, but was ignored.
https://github.com/ng-select/ng-select/issues/2155
In my opinion, this is an issue which should be fixed

There is also no workaround to override this attribute.

### Steps to Reproduce

1. Open the ng-select demo page: https://ng-select.github.io/ng-select/#/data-sources
2. Inspect ng-select input and realize that the attribute "autocomplete" is set with the dropDownId

![Image](https://github.com/user-attachments/assets/3d163e11-b112-46ba-bc78-22bbea835a1e)

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://ng-select.github.io/ng-select/#/data-sources

### Expected behaviour

The ng-select should have by default the autocomplete attribute set to "off"
A nice to have feature would be, that the autocomplete value can be passed as an attribute to the ng-select component.

### What version of the library you see this issue?

14.7.0

### What version of Angular are you using?

Angular 19
